multicluster/controllers/multicluster/commonarea/resourceimport_controller_test.go

```diff
@@ -491,3 +494,10 @@ func TestResourceImportReconciler_handleUpdateEvent(t *testing.T) {
 		})
 	}
 }
+
+func checkAnnotation(t *testing.T, svcImport *k8smcsapi.ServiceImport) {
+	id, ok := svcImport.Annotations[common.AntreaMCClusterIDAnnotation]
+	if id != localClusterID || !ok {
+		t.Errorf("latest ServiceImport annotation should be "+localClusterID+" but got %v", id)
```

Comment:
why don't use the expected cluster IP but an uncommon place holder?
```
t.Errorf("latest ServiceImport annotation should be %v but got %v", localClusterID, id)
```
Or
```
assert.Equal(t, localClusterID, id, "Unexpected ServiceImport clusterID annotation")
```

Reply:
done